### ISSUE TITLE
Link Power Bi to The program Icons on Left of the Screen

### ISSUE THREAD

**Issue by AMEENHashmi788** [kartoza/rir-dashboard#176]
![image](https://user-images.githubusercontent.com/92939482/151366803-dc84a9f8-4513-4b36-b4d9-d2a97bfb3d03.png)
[TABS LINK.pptx](https://github.com/kartoza/rir-dashboard/files/7950705/TABS.LINK.pptx)

**Comment by timlinux** [comment]
@AMEENHashmi788 yes we can add that - though currently they link to different end points depending on which scenario you are in (there are 4 endpoints per each of those icons). Were you thinking to do the same? Irwan will be offline tomorro, but once we have clarigty on the requirments we can follow up with it on monday.

**Comment by timlinux** [comment]
@AMEENHashmi788 you can alread set up these links but you need to use the django admin interface to do it:

https://staging.rir.kartoza.com/admin/rir_data/programinstance/

![image](https://user-images.githubusercontent.com/178003/151775418-debc6bf8-1cfc-4502-8eec-a64795838dc7.png)

@meomancer can you make the programme name the clickable link rather than the instance (basically swap the two columns around)

@AMEENHashmi788 in the detail view for the Programme Instance, you have four boxes where you can set a dashboard link to each scenario level.

![image](https://user-images.githubusercontent.com/178003/151775730-5286f455-9c99-4052-8f89-2564d90dc77e.png)

**Comment by timlinux** [comment]
Ok chatting to @AMEENHashmi788 he wants to keep the existing spreadsheet list but also have the ability to bring up a e.g. health dashboard (not needed on a per scenario button)

![image](https://user-images.githubusercontent.com/178003/151776652-a98e5de8-bb58-41a6-b4a4-d81ba2dc9770.png)
